app/presenters/post_presenter.rb

```diff
@@ -3,82 +3,93 @@ class PostPresenter
 
   attr_accessor :post, :current_user
 
-  def initialize(post, current_user = nil)
+  def initialize(post, current_user=nil)
     @post = post
     @current_user = current_user
   end
 
   def self.collection_json(collection, current_user)
-    collection.map {|post| PostPresenter.new(post, current_user)}
+    collection.map {|post| PostPresenter.new(post, current_user) }
   end
```

Comment:
If you want, you can inherit from BasePresenter, then you get the collection and initialize stuff "for free" :)